src/Program.ts

```diff
@@ -537,7 +554,14 @@ export class Program {
         let file = this.logger.time(LogLevel.debug, ['Program.setFile()', chalk.green(srcPath)], () => {
             //if the file is already loaded, remove it
             if (this.hasFile(srcPath)) {
-                this.removeFile(srcPath);
+                const existingFile = this.files[srcPath.toLowerCase()];
+                if (isBrsFile(existingFile)) {
+                    if (existingFile.fileContents === fileData.toString()) {
+                        // return;
+                    } else {
+                    }
+                }
+                this.removeFile(srcPath, true, true);
```

Comment:
Is this refactor still necessary? The commented-out `return` seems to be the only real reason for the change.